.github/workflows/prometheus-compliance-tests.yml

```diff
@@ -45,9 +45,8 @@ jobs:
         with:
           repository: prometheus/compliance
           path: compliance
-          ref: f0482884578bac67b053e3eaa1ca7f783d146557
       - name: Copy binary to compliance directory
-        run: mkdir compliance/remote_write_sender/bin && cp opentelemetry-collector-contrib/bin/otelcontribcol_linux_amd64 compliance/remote_write_sender/bin/otelcol_linux_amd64
+        run: mkdir compliance/remote_write_sender/bin && cp opentelemetry-collector-contrib/bin/otelcontribcol_linux_amd64 compliance/remote_write_sender/bin/otelcol_0.42.0_linux_amd64
```

Comment:
Can we add the reasoning on why we use this specific filename? It could be a bit confusing to someone without complete context. Is there a PR submitted in the compliance test repository to fix the hard coded file name? If so, can we also link that?

Reply:
Please add it as a comment above this run line in the GitHub actions file. It will make discovery much easier.